src/components/rules-creation-page/steps/InstanceSelectionStep.jsx

```diff
@@ -4,12 +4,8 @@ import { MultiSelector } from "d2-ui-components";
 import Instance from "../../../models/instance";
 
 export const getInstances = async d2 => {
-    const instances = await Instance.list(
-        d2,
-        { search: "" },
-        { page: 1, pageSize: 100, sorting: [] }
-    );
-    return instances.objects.map(instance => ({
+    const { objects } = await Instance.list(d2, null, { paging: false });
```

Comment:
[minor] The signature is `list(d2: D2, filters: TableFilters, pagination: TablePagination)`, we can pass `{}` as `filters`.